### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ericaig**
I made changes to the animateCSS function to accept multiple classes like 'tada faster'.

### CODE REVIEW

**Comment by WarenGonzaga** [comment]
@ericaig cool idea...

File: README.md [hunk 1]
```diff
@@ -106,17 +106,24 @@ You can use this simple function to add and remove the animations:
 
 ```javascript
 function animateCSS(element, animationName, callback) {
-    const node = document.querySelector(element)
-    node.classList.add('animated', animationName)
+    const node = document.querySelector(element);
+    let classes = animationName.split(" ");
+
+    classes.forEach((v,i)=>{
```
Comment (eltonmesquita) [inline README.md]: How about change the var `v` to a more descriptive name.
No need for the index here.

File: README.md [hunk 2]
```diff
@@ -106,17 +106,24 @@ You can use this simple function to add and remove the animations:
 
 ```javascript
 function animateCSS(element, animationName, callback) {
-    const node = document.querySelector(element)
-    node.classList.add('animated', animationName)
+    const node = document.querySelector(element);
```
Comment (eltonmesquita) [inline README.md]: Do we really need all those `;`? It's 2019 and it's pretty safe to omit those.

File: README.md (hunk 1 above)
Comment (eltonmesquita) [inline README.md]: Please format the code properly. Some spaces here would make it better...

File: README.md (hunk 2 above)
Comment (WarenGonzaga) [reply README.md]: Agree

**Comment by eltonmesquita** [comment]
I'm closing this as there's no response. A proper documentation will provide better ways to achieve this in the future.

**Pull request closed without merging**